sos/plugins/gluster.py

```diff
@@ -111,32 +113,31 @@ def setup(self):
         else:
             self.add_copy_spec("/var/log/glusterfs")
 
-        self.make_preparations(self.statedump_dir)
-        if self.check_ext_prog("killall -USR1 glusterfs glusterfsd"):
-            # let all the processes catch the signal and create statedump file
-            # entries.
-            time.sleep(1)
-            self.wait_for_statedump(self.statedump_dir)
-            self.add_copy_spec('/tmp/glusterdump.options')
-            self.add_copy_spec(self.statedump_dir)
-        else:
-            self.soslog.info("could not send SIGUSR1 to glusterfs processes")
+        if self.get_option("dump"):
+            self.make_preparations(self.statedump_dir)
+            if self.check_ext_prog("killall -USR1 glusterfs glusterfsd"):
+                # let all the processes catch the signal and create
+                # statedump file entries.
+                time.sleep(1)
+                self.wait_for_statedump(self.statedump_dir)
+                self.add_copy_spec('/tmp/glusterdump.options')
+                self.add_copy_spec(self.statedump_dir)
+            else:
+                self.soslog.info("could not send SIGUSR1 to glusterfs \
+                processes")
 
         volume_file = self.get_cmd_output_now("gluster volume info")
         if volume_file:
             for volname in self.get_volume_names(volume_file):
-                self.add_cmd_output([
-                                    "gluster volume geo-replication %s status"
-                                    % volname,
-                                    "gluster volume heal %s info" % volname,
-                                    "gluster volume heal %s info split-brain"
-                                    % volname,
-                                    "gluster snapshot list %s" % volname,
-                                    "gluster volume quota %s list" % volname,
-                                    "gluster volume rebalance %s status"
-                                    % volname,
-                                    "gluster snapshot info %s" % volname,
-                                    "gluster snapshot status %s" % volname])
+                self.add_cmd_output(
```

Comment:
Minor styling correction here. Put the starting bracket in the same line as `add_cmd_output` and the closing bracket and paren should be on a new line with the same indention as the call.

E.G.:
```
self.add_cmd_output([
    ...list...
])
```